src/service-worker/ServiceWorker.ts

```diff
@@ -873,7 +873,8 @@ export class ServiceWorker {
         If the permission is revoked, we should set the subscription state to permission revoked.
        */
       let subscriptionState: null | SubscriptionStateKind = null;
-      const pushPermission = await navigator.permissions.query({name:'push', userVisibleOnly:true});
+      const pushPermission: string = Notification["permission"];
```

Comment:
Now that we have the stub working can we make this `Notification.permission` and without the `string` cast.?

```typescript
const pushPermission = Notification.permission;
```